hera_cal/chunker.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2021 the HERA Project
+# Licensed under the MIT License
+
+
+from . import io
+import argparse
+import numpy as np
+import sys
+import warnings
+
+
+def chunk_data_files(filenames, inputfile, outputfile, chunk_size,
+                     polarizations=None, spw_range=None, throw_away_flagged_bls=False,
+                     clobber=False, ant_flag_yaml=None):
+    """A data file chunker
+
+    Parameters
+    ----------
+    filenames: list of strings
+        list of filenames to chunk. Should be homogenous in blt.
+    outpufile: str
+        name of outputfile to write time-concatenated data too.
+    polarizations: list of strs, optional
+        Limit output to polarizations listed.
+        Default None selects all polarizations.
+    spw_range: 2-list or 2-tuple
```

Comment:
This should specify the data type. `int` or `float` or...?

Reply:
they are supposed to be integers. Specified this in docstring.